packages/atomic/package.json

```diff
@@ -74,6 +74,8 @@
     "jest": "26.6.3",
     "jest-cli": "26.6.3",
     "postcss-import": "14.0.2",
+    "postcss-mixins": "^8.1.0",
```

Comment:
Since `@apply` no longer works inside `@screen`, I replaced it with `postcss-mixins`.